client/public/index.html

```diff
@@ -27,6 +27,7 @@
     <title>React App</title>
   </head>
   <body>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
```

Comment:
I guess this is related to the `react-owl-carousel` package and should be removed

Reply:
Thanks! Your are right!